pkg/clusteragent/clusterchecks/types/types.go

```diff
@@ -17,9 +17,25 @@ const (
 	ExtraHeartbeatLastChangeValue int64 = -1
 )
 
+/*
+NodeType represents the type of node reporting status to the cluster agent.
+It is used to distinguish between CLC runners, node agents, and unknown types.
+*/
+type NodeType uint8
+
+const (
+	// NodeTypeUnknown is used when the node type is not set or cannot be determined.
+	NodeTypeUnknown NodeType = 0
```

Comment:
I think this is never used in the code, because by looking at the function I suggested above, we should always be able to tell whether it’s a CLC or a node agent.